### DISCUSSION TITLE
new feature: ACTION_SEND intent for use by other apps

### DISCUSSION THREAD

**Discussion by tberghuis** [AppGround-io/bluetooth-keyboard-and-mouse-support#279] [Ideas]
I would pay for a premium feature that I could send keyboard text from other apps

**Comment by AppGround-io** [comment]
The latest version of the app (4.23.0) allows sending text from other apps directly to a specific device. For this to work, the device must be connected once before it appears in the sharing screen.

**Reply by tberghuis** [reply]
I meant programmatically not manually. Is that possible? If so, could you please share some example code?
